src/cc2olx/cli.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+
+from pathlib import Path
+
+RESULT_TYPE_FOLDER = "folder"
+RESULT_TYPE_ZIP = "zip"
+
+
+def parse_args(args=None):
+    parser = argparse.ArgumentParser(
+        description="This script converts imscc files into folders with "
+        "all the content; in the defined folder structure."
+    )
+    parser.add_argument(
+        "-i",
+        "--inputs",
+        nargs="*",
+        type=lambda p: Path(p).absolute(),
+        required=True,
+        help="Please provide the paths to the imscc files or directories "
+        "that contain them.",
+    )
+    parser.add_argument(
+        "-ll",
+        "--loglevel",
+        choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"],
+        default="INFO",
+        help="Please provide the appropriate level to change the detail of "
+        "logs. It can take one of the following values, DEBUG, INFO, WARNING, "
+        "ERROR, CRITICAL as argument.",
+    )
+    parser.add_argument(
```

Comment:
Is this still in progress? I don't see the portion that utilizes this setting.

Reply:
@kaizoku
>Is this still in progress?

Nope. There are no code that utilizes this setting in original repo. Should I implement it in this ticket?

Reply:
@0x29a, it'd be a bit weird to have the setting if it doesn't change anything. If we could implement this here it'd be nice.